testing/src/main/java/io/stargate/it/http/RestApiv2Test.java

```diff
@@ -538,7 +537,7 @@ public void createCustomIndex(CqlSession session) throws IOException {
     Map<String, String> optionsReturned = row.get().getMap("options", String.class, String.class);
 
     assertThat(optionsReturned.get("class_name")).isEqualTo(indexType);
-    assertThat(optionsReturned.get("target")).isEqualTo("\"lastname\"");
+    assertThat(optionsReturned.get("target")).isEqualTo("lastname");
```

Comment:
That one puzzles me. I'm pretty sure unquoted is the right answer, that's what I get when creating the index via cqlsh. I don't know how it can end up quoted in the v1 test.

Reply:
Yeah quoted seems wrong... unless it'd be for reason like retaining casing?

@jeffreyscarpenter Do you think this might have been `"lastName"` in SGv1, changed to all lower-case for SGv2?

Reply:
Right, it was mixed case on `master`.
I've reverted to the original code, the v2 implementation handles case the same.